### PR TITLE
Selection rectangle is drawn after exiting radar widget while dragging the viewport

### PR DESCRIPTION
**Pull request by hktk1f40**
See gifs below for mentioned effect and changed behavior

Before:
https://www.dropbox.com/s/6la0yq5w1n6cv0u/ra-before-fix.gif?dl=0

After:
https://www.dropbox.com/s/wjk0n15t8l3p6cr/ra-after-fix.gif?dl=0

### CODE REVIEW

**Comment by Mailaender** [comment]
:+1: I can confirm this fixes https://github.com/OpenRA/OpenRA/issues/2793 (tested the ra and d2k mod).

File: OpenRA.Game/Widgets/WorldInteractionControllerWidget.cs [hunk 1]
```diff
@@ -85,33 +86,40 @@ public override bool HandleMouseInput(MouseInput mi)
...
 			// don't issue orders while selecting
 			if (mi.Button == MouseButton.Right && mi.Event == MouseInputEvent.Down && !hasBox) 
 				ApplyOrders(World, xy, mi);
 
+			lastMousePosition = xy;
+
 			return true;
 		}
 
+		bool IsDraging
```
Comment (obrakmann) [inline OpenRA.Game/Widgets/WorldInteractionControllerWidget.cs]: -> IsDragging (double g)

**Comment by obrakmann** [comment]
:+1: after the spelling error has been corrected

**Comment by atimoschenkow** [comment]
Fixed typo.

**Comment by obrakmann** [comment]
Thanks!

[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/638c377176189355212bbe6b1161c350c523ee7e%5E...638c377176189355212bbe6b1161c350c523ee7e)

**Pull request merged**